nisystemlink/clients/auth/models/_additional_models.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+from typing import Dict, List, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+from pydantic import Field
+
+from ._auth_models import AuthPolicy, Error, Org, User, Workspace
+
+
+class AuthResponse(JsonModel):
+    """Information about the authenticated caller."""
+
+    user: Optional[User]
+    """Details of authenticated caller"""
+    org: Optional[Org]
+    """Organization of authenticated caller"""
+    workspaces: Optional[List[Workspace]]
+    """List of workspaces the authenticated caller has access"""
+    policies: Optional[List[AuthPolicy]]
+    """List of policies for the authenticated caller"""
+    properties: Optional[Dict[str, str]] = Field(None, example={"key1": "value1"})
+    """A map of key value properties"""
+    error: Optional[Error]
```

Comment:
It doesn't make sense to have the error field here. The HTTP API will return either the auth information OR an error, but not both in the same JSON body.